src/tools/genpolicy/src/config_map.rs

```diff
@@ -124,6 +125,7 @@ impl yaml::K8sResource for ConfigMap {
         &self,
         _policy_mounts: &mut Vec<policy::KataMount>,
         _storages: &mut Vec<agent::Storage>,
+        _persistent_volume_claims: &Vec<PersistentVolumeClaim>,
```

Comment:
I prefer the older convention, that shows more clearly which *.rs file a struct name is coming from. 

I would use pvc.rs and pvc::PersistentVolumeClaim instead of just PersistentVolumeClaim.

Reply:
Gotcha - any objection to renaming `persistent_volume_claim.rs` to `pvc.rs` then?

Reply:
Please rename it that way.

Reply:
Done